packages/renderer/src/lib/table/Table.svelte

```diff
@@ -66,6 +67,17 @@ function sort(column: Column<any>) {
   data = data.sort(comparator);
 }
 
+onMount(async () => {
+  if (defaultSortColumn) {
+    for (const column of columns) {
+      if (column.title === defaultSortColumn) {
+        sortCol = column;
+        sortAscending = true;
+      }
+    }
+  }
+});
+
```

Comment:
it does the job but I'm wondering if we could not use some filters/find/foreach/map/etc keywords to to the same without having 3 sub-blocks

Reply:
Changed to use .find() in new commit.